src/System.Windows.Forms/src/System/Windows/Forms/ContainerControl.cs

```diff
@@ -1170,6 +1171,15 @@ internal override void Scale(SizeF includedFactor, SizeF excludedFactor, Control
             }
         }
 
+        /// <summary>
+        /// Scales container's properties Min and Max size with the scale factor provided.
+        /// </summary>
+        /// <param name="xScaleFactor">Scale factor to be applied on width of the property being scaled</param>
+        /// <param name="yScaleFactor">Scale factor to be applied on height of the property being scaled</param>
+        /// <param name="updateContainerSize">Specifies if container size need to be updated along with properties being scaled</param>
```

Comment:
```suggestion
        /// <param name="xScaleFactor">The scale factor to be applied on width of the property being scaled.</param>
        /// <param name="yScaleFactor">The scale factor to be applied on height of the property being scaled.</param>
        /// <param name="updateContainerSize"><see langword="true"/> to resize of the container control along with properties being scaled; otherwise, <see langword="false"/>.</param>
```

I think there's a Roslyn-based doc analyzer that we haven't enabled yet to track the full stops :)